src/bp/bootstrap.ts

```diff
@@ -91,31 +91,34 @@ async function resolveModules(moduleConfigs: ModuleConfigEntry[], resolver: Modu
   return { loadedModules, erroredModules }
 }
 
-async function prepareLocalModules(logger: sdk.Logger) {
-  if (!Ghost.useDbDriver) {
+async function prepareLocalModules(app: BotpressApp, logger: sdk.Logger) {
+  if (!app.ghost.useDbDriver) {
     return
   }
 
   try {
     // We remove the local copy in case something is deleted from the database
-    await Ghost.root(false).deleteFolder('modules')
+    await app.ghost.root(false).deleteFolder('modules')
   } catch (err) {
     logger.attachError(err).warn('Could not clear local modules cache')
   }
 
-  await Ghost.root().syncDatabaseFilesToDisk('modules')
+  await app.ghost.root().syncDatabaseFilesToDisk('modules')
 }
 
 async function start() {
-  await setupDebugLogger()
-
   if (cluster.isMaster) {
+    const loggerProvider = createLoggerProvider()
+    await setupDebugLogger(loggerProvider)
     // The master process only needs getos and rewire
-    return setupMasterNode(await getLogger('Cluster'))
+    return setupMasterNode(await getLogger(loggerProvider, 'Cluster'))
   }
```

Comment:
This is the part of the code that was causing all the services to be initialized twice. Like the comment says 
> The master process only needs getos and rewire
But because this file was including the app.ts file, it was initializing the server as well for no reason